Patches/ChatCommandPatch.cs

```diff
@@ -2428,8 +2428,26 @@ public static void Postfix(ChatController __instance)
         if (player == null) return;
         
         (string msg, byte sendTo, string title) = Main.MessagesToSend[0];
+
+        if (sendTo != byte.MaxValue)
+        {
+            if (Utils.GetPlayerInfoById(sendTo) != null)
+            {
+                var targetinfo = Utils.GetPlayerInfoById(sendTo);
+
+                if (targetinfo.DefaultOutfit.ColorId == -1)
```

Comment:
-1 is fortegreen color?

Reply:
Uhh, i see
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/4c73d3cd-4466-4ae2-afdb-1d80b210294c)